src/pages/styles/icons.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { CSSProperties } from 'react';
+import { graphql } from 'gatsby';
+import styled from 'styled-components';
+import { NamespacesConsumer } from 'react-i18next';
+import { withI18next } from '@wapps/gatsby-plugin-i18next';
+import { suomifiTheme, Icon } from 'suomifi-ui-components';
+import {
+  allIcons,
+  IconKeys,
+  allStaticIcons,
+  StaticIconKeys,
+} from 'suomifi-icons';
+
+import Layout from 'components/layout';
+import SEO from 'components/seo';
+import Annotation from 'components/Annotation';
+import ComponentDescription from 'components/ComponentDescription';
+import sideNavData from 'config/sidenav/styles';
+import ComponentExample from 'components/ComponentExample';
+import { Heading, Text, Paragraph } from 'components/ResponsiveComponents';
+import { ReactComponent as Slack } from 'icons/slack.svg';
+
+const iconCategories = [
+  {
+    id: 'baseIcons',
+    icons: allIcons,
+  },
+  {
+    id: 'illustrativeIcons',
+    icons: allStaticIcons.filter(icon => icon !== 'toggle'),
+  },
+];
+const StyleIcon = styled(Icon)`
+  height: 1.5em;
+  width: 1.5em;
+`;
+
+const getExampleIcon = (
+  id: string,
+  label: string,
+  style?: CSSProperties,
+): JSX.Element => (
+  <div
+    key={id}
+    style={{
+      margin: `${suomifiTheme.spacing.s} ${suomifiTheme.spacing.xl} ${
+        suomifiTheme.spacing.l
+      } 0`,
+      lineHeight: '1rem',
```

Comment:
In the design the font is specified as 14px semibold, but on the page it is 18px at the moment.